src/workspacer/workspacer.csproj

```diff
@@ -1,35 +1,41 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
     <TargetFramework>net5.0-windows</TargetFramework>
-    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
+    <RuntimeIdentifier>win10-x64</RuntimeIdentifier>
     <UseWindowsForms>true</UseWindowsForms>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <ApplicationIcon>logo.ico</ApplicationIcon>
+    <Description>a tiling window manager for Windows</Description>
+    <Authors>Rick Button</Authors>
+    <Version>0.9.9</Version>
   </PropertyGroup>
-
   <ItemGroup>
     <EmbeddedResource Include="Config\workspacer.config.template.csx" />
   </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="Autoupdater.NET.Official" Version="1.6.4" NoWarn="NU1701" />
     <PackageReference Include="Microsoft.CodeAnalysis.Scripting" Version="3.8.0" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="Octokit" Version="0.48.0" />
   </ItemGroup>
-
-  <Target Name="PortablePostBuild" AfterTargets="Build">
+  <Target Name="CopyPluginsAfterBuild" AfterTargets="Build">
```

Comment:
Target**s** is plural. I believe you should be able to use the same target-block for both Build and Publish to avoid code-duplication.

https://docs.microsoft.com/en-us/visualstudio/msbuild/target-element-msbuild?view=vs-2019

Reply:
According to the docs, it would run "after the specified target or targets", which means it would only run when both build and publish runs. Also, the directory of the build and publish folders are different ( `TargetDir` vs `PublishDir`).

Reply:
You're right. I missed that. 

Let's keep it as is then.